crates/util/python-utils/Cargo.toml

```diff
@@ -21,7 +21,7 @@ bdk_wallet = "1.0.0-beta.5"
 # "abi3-py310" tells pyo3 (and maturin) to build using the stable ABI with minimum Python version 3.10
 musig2.workspace = true
 pyo3 = { version = "0.22.6", features = ["extension-module", "abi3-py310"] }
-secp256k1 = { workspace = true, features = ["rand-std"] }
+secp256k1 = { workspace = true, features = ["rand-std", "global-context"] }
```

Comment:
We don't need this because we have `bdk_wallet::bitcoin::secp256k1` re-exported from `bdk`'s `bitcoin` and then `bitcoin`'s `secp256k1`

Reply:
Yes, I understand this and this is what surprises me. But the `python-utils` strangely stops compiling if I remove this as a _direct_ dependency (**even though it's not used it as a DIRECT import**, but through the chain of re-imports).

Even though I understand it's not ideal, I propose for the middleground: I'll leave TODO in `python-utils/cargo.toml` to get rid of this and we can figure it out later (ideally together, please)?

Reply:
Ok we can debug this later.
To be honest, I don't want to have to use Python for func-tests.
So in the future all of this crate's code would be migrated over to a Rust func-test harness somehow.